code/game/objects/items/grenades/grenade.dm

```diff
@@ -150,7 +150,7 @@
 
 /obj/item/grenade/hit_reaction(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK)
 	var/obj/projectile/P = hitby
-	if(damage && attack_type == PROJECTILE_ATTACK && P.damage_type != STAMINA && prob(15))
+	if(damage && attack_type == PROJECTILE_ATTACK && P.damage_type != STAMINA && prob(1) && prob(50))
```

Comment:
```suggestion
	if(damage && attack_type == PROJECTILE_ATTACK && P.damage_type != STAMINA && prob(0.5))
```

Reply:
Oh, thank you! I wasn't sure if the argument only took integers or if it took floats, so I just did it like this to ensure that it worked. I'll fix this, thank you.

Reply:
The modification should be complete now